packages/core/src/amazonq/lsp/lspController.ts

```diff
@@ -183,4 +182,12 @@ export class LspController {
             }
         })
     }
+
+    private async setupLsp(context: vscode.ExtensionContext) {
+        await lspSetupStage('all', async () => {
+            const installResult = await new WorkspaceLSPResolver().resolve()
+            await lspSetupStage('launch', async () => activateLsp(context, installResult.resourcePaths))
+            getLogger().info('LspController: LSP activated')
```

Comment:
Not necessarily an issue for this PR, but I think we should probably scope these messages to regular codewhisperer language server vs workspace context one

Reply:
Yeah, I think we could make use of the logging header here that references which LSP it is.